addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -103,6 +110,47 @@ export class WebsiteLoader extends Component {
         });
     }
 
+    /**
+     * Initializes the progress bar.
+     */
+    initProgressBar() {
+        if (this.updateProgressInterval) {
+            return;
+        }
```

Comment:
`updateProgressInterval` is never set back to a falsy value, so this effectively mean that once `initProgressBar` is called, recalling it never does anything again. Is this what you wanted? If yes, should we not just prevent recalling it again if already called?

Reply:
Yes that's what I want. When there are modules to install, the progress bar is started by the `trackModules` function. As it is a recursive function, I did this simple solution (if the progress bar has been started, it's not possible to start it again). If you prefer I can move this logic in `trackModules` ? @qsm-odoo

Reply:
Can't you call `initProgressBar` where `trackModules` is called instead?

Reply:
No because `this.featuresInstallInfo` is computed inside the `trackModules` function and the progress bar needs `this.featuresInstallInfo`.